### PR TITLE
Feat: Project Euler Problem 14

### PR DESCRIPTION
**Pull request by domix80**
Adding the resolution for Euler Problem 14: Longest Collatz sequence

### CODE REVIEW

**Comment by domix80** [comment]
@aparibocci, Could you please review it?

**Comment by domix80** [comment]
@StepfenShawn - May I ask you to review and eventually merge the code?

**Review by StepfenShawn — CHANGES_REQUESTED** [review]
Please don't update the `Directory.md `. It's updated automatically.

File: project_euler/problem_014/sol1.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      index_best_result = index_candidate
+      value = num
+    end
+  end
+  result = value
+end
+
+answer = solution()
+p answer
+
```
Comment (StepfenShawn) [inline project_euler/problem_014/sol1.rb:42]:
```suggestion
```

**Review by StepfenShawn — CHANGES_REQUESTED** [review]

**Comment by domix80** [comment]
Hi @StepfenShawn ,

Thanks for the review.
I have restored director.md file

Br,
Domenico

**Review by StepfenShawn — APPROVED** [review]

**Pull request merged**